tst/EnergyPlus/unit/WindowManager.unit.cc

```diff
@@ -2836,6 +2836,8 @@ TEST_F(EnergyPlusFixture, WindowManager_SrdLWRTest)
     state->dataScheduleMgr->Schedule(1).CurrentValue = 25.0; // Srd Srfs Temp
     // Calculate temperature based on supply flow rate
 
+    HeatBalanceSurfaceManager::InitSurfacePropertyViewFactors(*state);
+
     WindowManager::CalcWindowHeatBalance(*state, surfNum2, state->dataHeatBalSurf->SurfHConvInt(surfNum2), inSurfTemp, outSurfTemp);
```

Comment:
Need to reset the sky and ground surface view factors for exterior surface. This unit test failed due to deletion of the redundant code.